.vscode/launch.json

```diff
@@ -1,22 +1,22 @@
 {
-    // Use IntelliSense to learn about possible attributes.
```

Comment:
Probably revert this file. If you have auto-fix linting on save enabled. You can either save without formatting via the VSCode command `File: Save without formatting`, or just don't stage this file.

Reply:
reverted!